api/src/pcapi/tasks/external_api_booking_notification_tasks.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from pcapi.routes.serialization import BaseModel
+from pcapi.tasks.decorator import task
+from pcapi.tasks.serialization.external_api_booking_notification_tasks import ExternalApiBookingNotificationRequest
+from pcapi.utils import requests
+
+from . import settings
+
+
+logger = logging.getLogger(__name__)
+
+
+class ExternalApiBookingNotificationTaskPayload(BaseModel):
+    data: ExternalApiBookingNotificationRequest
+    notificationUrl: str
+
+
+@task(settings.GCP_EXTERNAL_API_BOOKING_NOTIFICATION_QUEUE_NAME, "/external_api/booking_notification")
+def external_api_booking_notification_task(payload: ExternalApiBookingNotificationTaskPayload) -> None:
+    try:
+        requests.post(payload.notificationUrl, json=payload.data)
+    except requests.ExternalAPIException as exception:
```

Comment:
Cette exception n'est pas levée par `post()`. Pour faire simple, tu peux `except requests.exceptionsRequestException` ([cf. doc](https://requests.readthedocs.io/en/latest/user/quickstart/?highlight=raise_for_status()#errors-and-exceptions)).

Par contre, cela ne prendra en compte que les erreurs réseau (timeout, DNS, etc.). Aucune exception n'est levée si on reçoit bien une réponse, mais que son statut HTTP n'est pas une 20x. Or on aimerait certainement logger aussi ces erreurs, j'imagine. Pour cela, tu peux appeler `response.raise_for_status()` ([cf. doc](https://requests.readthedocs.io/en/latest/user/quickstart/?highlight=raise_for_status()#response-status-codes)).